packages/camera/camera/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.10.0
+
+* **Breaking Change** Bumps default camera_web package version to 0.3.0.
```

Comment:
how is this a breaking change?

Reply:
@blasten because 0.3.0 is a breaking change on camera_web. We should probably provide some details here. Something like: 

```
**BREAKING CHANGE** Bumps default camera_web package version to 0.3.0 which renames permission exception code from `cameraPermission` to `CameraAccessDenied`. 
```